docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -42,6 +42,9 @@ For these platforms, the platform alone determines the browser that is used.
 |iOS|any|Safari with WKWebView|
 |Android|any|Chrome|
 
+> [!IMPORTANT]
+> [Conditional Access](/azure/active-directory/conditional-access/overview) is not supported for Office Add-ins on iOS. Those add-ins use embedded Safari, not Edge.
```

Comment:
Could you say WKWebView instead of "embedded Safari"? Or maybe "WKWebView based on Safari".

Reply:
Sure. How about "Those add-ins use the Safari-based WKWebView, not an Edge-based browser control."